{
    "sourceFile": "node_modules/next/dist/esm/client/components/is-next-router-error.js.map",
    "activeCommit": 0,
    "commits": [
        {
            "activePatchIndex": 0,
            "patches": [
                {
                    "date": 1746892765806,
                    "content": "Index: \n===================================================================\n--- \n+++ \n"
                }
            ],
            "date": 1746891704042,
            "name": "cart",
            "content": "{\"version\":3,\"sources\":[\"../../../src/client/components/is-next-router-error.ts\"],\"sourcesContent\":[\"import {\\n  isHTTPAccessFallbackError,\\n  type HTTPAccessFallbackError,\\n} from './http-access-fallback/http-access-fallback'\\nimport { isRedirectError, type RedirectError } from './redirect-error'\\n\\n/**\\n * Returns true if the error is a navigation signal error. These errors are\\n * thrown by user code to perform navigation operations and interrupt the React\\n * render.\\n */\\nexport function isNextRouterError(\\n  error: unknown\\n): error is RedirectError | HTTPAccessFallbackError {\\n  return isRedirectError(error) || isHTTPAccessFallbackError(error)\\n}\\n\"],\"names\":[\"isHTTPAccessFallbackError\",\"isRedirectError\",\"isNextRouterError\",\"error\"],\"mappings\":\"AAAA,SACEA,yBAAyB,QAEpB,8CAA6C;AACpD,SAASC,eAAe,QAA4B,mBAAkB;AAEtE;;;;CAIC,GACD,OAAO,SAASC,kBACdC,KAAc;IAEd,OAAOF,gBAAgBE,UAAUH,0BAA0BG;AAC7D\"}"
        }
    ]
}